src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const head = document.querySelector('thead');
+const body = document.querySelectorAll('tbody tr');
+const tbody = document.querySelector('tbody');
+
+function converter(Salary) {
```

Comment:
с большой буквы только классы или конструкторы

```suggestion
function convertSalary(salary) {
```